pkg/proc/native/proc_windows.go

```diff
@@ -300,6 +301,10 @@ func (dbp *Process) waitForDebugEvent(flags waitForDebugEventFlags) (threadID, e
 		case _RIP_EVENT:
 			break
 		case _EXCEPTION_DEBUG_EVENT:
+			if flags&waitContinueAllExceptions != 0 {
+				continueStatus = _DBG_CONTINUE
```

Comment:
maybe

s/_DBG_CONTINUE/_DBG_EXCEPTION_NOT_HANDLED /

because _DBG_EXCEPTION_NOT_HANDLED will exit process regardless of situation.